### PR TITLE
Updated roadmap 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated roadmap

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
 * First Class Client
-* iBFT 2.0
 
-### Permissioning
-We are implementing the key elements of an Enterprise Ethereum Permissioned network.  The initial version of this will be based around a JSON RPC API to manage the network. This will form the foundation for a smart contract based solution which will be developed in the `next` release (1.1)
+### Privacy
+The Enterprise Ethereum `restricted` privacy will be implemented.
+
+### Smart Contract Based Permissioning
+Building on the Permissioning system  implemented in version 1.0 of Pantheon, we will use a smart contract to share this information across the network, giving a consistent set of permissions, and ensuring that all nodes in the network work consistently.
```
Comment (arash009) [inline ROADMAP.md]: How about:
"Building on the Permissioning system implemented in version 1.0 of Pantheon, we will use a smart contract to share this information across the network, creating a single, decentralized source of truth for node and account level permission data."

File: ROADMAP.md [hunk 2]
```diff
@@ -1,39 +1,42 @@
...
+evolve and change over time.  In particular the features in later versions are likely to be refined and change.
 
-We use the approach of  `#now`, `#next`, `#later` [used by foursquare](https://medium.com/@noah_weiss/now-next-later-roadmaps-without-the-drudgery-1cfe65656645), with a slightly different time horizon.  Our now scale is about 3 months, next about 6 months, and then later is beyond.
+We use the approach of  `#now`, `#next`, `#later` [used by foursquare](https://medium.com/@noah_weiss/now-next-later-roadmaps-without-the-drudgery-1cfe65656645), with a slightly different time horizon.  
+Our now scale is about 3 months, next about 6 months, and then later is beyond.
 
-## Now
-Our key three areas for now are:
-* Permissioning
+## Now/v1.1
```
Comment (timbeiko) [inline ROADMAP.md]: Nit, but perhaps `Now (up to v1.1)`, thoughts?

File: ROADMAP.md [hunk 3]
```diff
@@ -1,39 +1,42 @@
...
+
+### Smart Contract Based Permissioning
+Building on the Permissioning system  implemented in version 1.0 of Pantheon, we will use a smart contract to share this information across the network, giving a consistent set of permissions, and ensuring that all nodes in the network work consistently.
 
 ### First Class Client
-There is an ongoing piece of work underway enhancing the core performance of Pantheon, and ensuring that it behaves well as a first class client. The key elements of this are implementation of some performance benchmarks, finalising the options for the command line, and implementing an appropriate fast sync mechanism.
-
-### iBFT 2.0
-Work is underway designing and implementing a new PBFT consensus mechanism, improving the existing iBFT protocol, addressing some liveness and safety issues in the original protocol.
+There is an ongoing piece of work underway enhancing the core performance of Pantheon, and ensuring that it behaves well as a first class client. The key elements of this are implementation of some performance benchmarks and enhancing our fast sync mechanism.
```
Comment (timbeiko) [inline ROADMAP.md]: How about:
> There is ongoing work to enhance Pantheon's performance, to ensure it behaves well as a first class client. The current initiatives include the implementation of performance benchmarks, of a fast sync mechanism and of support for all major testnets.

File: ROADMAP.md (hunk 1 above)
Comment (MadelineMurray) [reply ROADMAP.md]: Updated

File: ROADMAP.md (hunk 2 above)
Comment (MadelineMurray) [reply ROADMAP.md]: Makes sense. Updated

File: ROADMAP.md (hunk 3 above)
Comment (MadelineMurray) [reply ROADMAP.md]: Updated

**Pull request merged**